src/server/pfs/cmds/cmds.go

```diff
@@ -1924,15 +2043,17 @@ $ {{alias}} foo@master:path1 bar@master:path2`,
 		}),
 	}
 	deleteFile.Flags().BoolVarP(&recursive, "recursive", "r", false, "Recursively delete the files in a directory.")
-	deleteFile.Flags().StringVar(&project, "project", project, "Project in which repo is located.")
+	deleteFile.Flags().StringVar(&project, "project", project, "Specify the project (by name) where the file's repo is located.")
 	shell.RegisterCompletionFunc(deleteFile, shell.FileCompletion)
 	commands = append(commands, cmdutil.CreateAliases(deleteFile, "delete file", files))
 
+	// OBJECT COMMANDS
+
 	objectDocs := &cobra.Command{
```

Comment:
I didn't think we supported object APIs and more.

Reply:
I think you are right, that was the only one there. I can look into removing it after we get these changes in!